From: Robert Ancell <robert.ancell@canonical.com>
Date: Tue, 21 Sep 2021 15:24:20 +1200
Subject: Fix documentation for g_dbus_object_manager_get_object().

The class in the documentation is true for GDBusObjectManagerClient, but not for GDBusObjectManagerServer.

Origin: upstream, 2.70.1, commit:f86488c1e6559e8488c015e04006ad5ca54051dc
---
 gio/gdbusobjectmanager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/gdbusobjectmanager.c b/gio/gdbusobjectmanager.c
index 169e7d1..4c4c74b 100644
--- a/gio/gdbusobjectmanager.c
+++ b/gio/gdbusobjectmanager.c
@@ -211,7 +211,7 @@ g_dbus_object_manager_get_objects (GDBusObjectManager *manager)
  * @manager: A #GDBusObjectManager.
  * @object_path: Object path to look up.
  *
- * Gets the #GDBusObjectProxy at @object_path, if any.
+ * Gets the #GDBusObject at @object_path, if any.
  *
  * Returns: (transfer full) (nullable): A #GDBusObject or %NULL. Free with
  *   g_object_unref().
